srcpkgs/python3-scipy/template

```diff
@@ -1,25 +1,52 @@
 # Template file for 'python3-scipy'
 pkgname=python3-scipy
-version=1.3.2
-revision=2
+version=1.5.0
+revision=1
 wrksrc="scipy-${version}"
 build_style=python3-module
-pycompile_module="scipy"
 make_check_args="--force"
-hostmakedepends="python3-setuptools python3-Cython gcc-fortran"
-makedepends="python3-devel python3-numpy lapack-devel"
+hostmakedepends="python3-setuptools python3-Cython python3-numpy gcc-fortran"
+makedepends="python3-devel python3-numpy python3-pybind11 lapack-devel openblas-devel"
 depends="python3-numpy"
 checkdepends="python3-nose"
 short_desc="Scientific library for Python3"
 maintainer="Alessio Sergi <al3hex@gmail.com>"
 license="BSD-3-Clause"
 homepage="https://scipy.org/scipylib/"
 distfiles="https://github.com/scipy/scipy/releases/download/v${version}/scipy-${version}.tar.xz"
-checksum=1d2f09bcb6c4b66a65d9f49d21fa065f5396c940edac8285b87947b8d21b55f8
+checksum=23baeaa18803d12d1abdff3f5c148b1085c2dc4028c6b8efce652dde2119b41c
 
-nocross="https://build.voidlinux.org/builders/aarch64_builder/builds/24733/steps/shell_3/logs/stdio"
+if [ "$CROSS_BUILD" ]; then
+	# Make sure numpy is found for the target arch first
+	CFLAGS+=" -I${XBPS_CROSS_BASE}${py3_sitelib}/numpy/core/include"
+	LDFLAGS+=" -L${XBPS_CROSS_BASE}${py3_sitelib}/numpy/core/lib"
 
-LDFLAGS+="-shared"
+	# Tell numpy.distutils where to find FORTRAN compilers
+	export F77="${XBPS_CROSS_TRIPLET}-gfortran"
+	export F90="${XBPS_CROSS_TRIPLET}-gfortran"
```

Comment:
Should we add those variables into `xbps-src` instead?

Reply:
I envision a `numpy` build helper that sets the environment and adds the right `CFLAGS` and `LDFLAGS` search paths to find the target numpy libs. That might help cross compile things like scikit that currently break. Outside of this case, I'm not sure we have any need to point to FORTRAN compilers.

These improvements will require some more thought and testing and don't need to hold up scipy.

Reply:
A closer look shows that `xbps-src` sets `FC` for the target. I've modified the template to use this variable but still think that the `F90` and `F77` are special enough cases that they should be put here or in a `numpy` build style rather than in the universal environment.